src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -35,7 +35,378 @@ import DeploymentConfigToolbar from './DeploymentTemplateView/DeploymentConfigTo
 import { SaveConfirmationDialog, SuccessToastBody } from './DeploymentTemplateView/DeploymentTemplateView.component'
 import { deploymentConfigReducer, initDeploymentConfigState } from './DeploymentConfigReducer'
 import DeploymentTemplateReadOnlyEditorView from './DeploymentTemplateView/DeploymentTemplateReadOnlyEditorView'
+import { stat } from 'fs'
 
+export const dummy = {
```

Comment:
if possible move it into separate file named constants

Reply:
no